src/client/components/EditUserForm/EditUserForm.tsx

```diff
@@ -12,6 +12,7 @@ import { useUser } from 'client/store/user'
 import { useCountryIso, useOnUpdate } from 'client/hooks'
 import DisableUser from 'client/components/EditUserForm/DisableUser'
 import UserCountryRoleSelector from 'client/components/EditUserForm/UserCountryRoleSelector'
+import { useOptionsAppellation } from 'client/pages/Section/Contacts/hooks/useOptionsAppellation'
```

Comment:
Should this hook be a more generic hook?

Reply:
Is this used somewhere else in same format?

Reply:
Yes, it's used in `useColumns` for Contacts, in the same format { label, value }.

Reply:
If we make it more generic, where would be a good place to put it? `src/client/hooks` doesn't seem like is the right place, or is it?

Reply:
Moved it 👍